service_capacity_modeling/hardware/profiles/shapes/aws/auto_m6i.json

```diff
@@ -15,7 +15,7 @@
       "cpu": 4,
       "cpu_cores": 2,
       "cpu_ghz": 3.5,
-      "cpu_ipc_scale": 1.0,
+      "cpu_ipc_scale": 1.15,
```

Comment:
Oh interesting, I thought all of the 15% uplift from m5 to m6i was just the 3.1 -> 3.5. There was a 30% lift from 5th to 6th?

Reply:
Dependent on workload, yes. Ice Lake has a significant IPC bump for many workloads (especially crypto and AVX-512). The turbo curve also shifted; the sustained all-core turbo on m6i is quite flat with sustained all-core at 3.5, which is a difference from m5.

Reply:
I'm inclined to leave these ipc_scale numbers alone unless we change all of the (m|r|c)6i and bump the m7i at the same time or base them all on an IPC benchmark.

We really just added that scale factor to account for m7a cores being significantly faster and to make it so m7i (3.2G clock) didn't come out as _slower_ than m6i (3.5G clock). With this change the 6th gen is now faster again (3.5 * 1.15 = 4.025 effective versus 7th gen 3.2 * 1.25 = 4 effective).

We could also split this into two parts, one that adds the shapes without making an IPC claim and the other that refactors _all_ affected shapes in the 6th gen.